### PR TITLE
feat(profiling) use continuous profiler instead of the span/txn based one

### PR DESCRIPTION
**Pull request by JonasBa**
We want to start ingesting some profiles from a production env using the new continuous profiler

### CODE REVIEW

File: src/instrument.ts [hunk 1]
```diff
@@ -4,6 +4,16 @@ import {nodeProfilingIntegration} from '@sentry/profiling-node';
 Sentry.init({
   dsn: process.env.SENTRY_DSN,
   integrations: [nodeProfilingIntegration()],
-  profilesSampleRate: 1,
   tracesSampleRate: 1,
 });
+
+const client = Sentry.getClient();
+
+if(client) {
```
Comment (AbhiPrasad) [inline src/instrument.ts:12]:
```suggestion
if (client) {
```

File: src/instrument.ts [hunk 2]
```diff
@@ -4,6 +4,16 @@ import {nodeProfilingIntegration} from '@sentry/profiling-node';
...
+  const profilingIntegration = client.getIntegrationByName("ProfilingIntegration");
+
+  if(profilingIntegration) {
```
Comment (AbhiPrasad) [inline src/instrument.ts:15]:
```suggestion
  if (profilingIntegration) {
```

File: src/instrument.ts [hunk 3]
```diff
@@ -4,6 +4,16 @@ import {nodeProfilingIntegration} from '@sentry/profiling-node';
...
+    // @ts-expect-error this is purposefuly not exposed by the SDK for now
+    profilingIntegration._profiler.start();
```
Comment (AbhiPrasad) [inline src/instrument.ts:17]: Oh interesting, this doesn't require a `profilesSampleRate`?

**Review by AbhiPrasad — APPROVED** [review]

File: src/instrument.ts (hunk 3 above)
Comment (JonasBa) [reply src/instrument.ts:17]: No, we actually rely on it not being set. It's a bit weird from the API pov, but we dont want to apply the sampling rate per chunk.

**Pull request merged**